src/DevBetterWeb.Web/Pages/User/MyProfile/Billing.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@page
+@model DevBetterWeb.Web.Pages.User.MyProfile.BillingModel
+@{
+}
+
+<h3>My Profile: Billing History</h3>
+
+<div class="row">
+    <div class="col-7">
+        <div class="card">
+            <div class="card-header">
+                Overview
+            </div>
+            <div class="card-body">
+                <p>@Model.GetOverviewBody()</p>
+            </div>
+        </div>
+        <br />
+        @if (!Model.GetIsAlumni())
```

Comment:
Don't base this on alumni status, but on whether they have an active subscription to cancel.